proto/panacea/datadeal/v2alpha2/deal.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package panacea.datadeal.v2alpha2;
+
+option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// Deal defines a deal.
+message Deal {
+  uint64 deal_id = 1;
+  string deal_address = 2;
+  repeated string data_schema = 3;
+  cosmos.base.v1beta1.Coin budget = 4;
+  uint64 max_num_data = 5;
+  uint64 cur_num_data = 6;
+  string owner_address = 7;
```

Comment:
Why don't we change to `buyer_address`? It is clearer than 'owner' I think as we discussed before

Reply:
Ah... this might be missed when reverting it. I'll change it!